src/partials/head-meta-opengraph.hbs

```diff
@@ -0,0 +1,9 @@
+<!-- START Facebook OpenGraph tags -->
```

Comment:
Please remove these comments, they are also visibe in the source code and IMHO this is absolutely not needed, these tags are pretty much wellknown.